OWL2/OWL22CASL.hs

```diff
@@ -118,7 +118,21 @@ uriToIdM = return . uriToCaslId
 
 -- | Extracts Id from URI
 uriToCaslId :: IRI -> Id
-uriToCaslId urI =  stringToId $ showIRI urI
+uriToCaslId urI = 
```

Comment:
I think this function is not specific to this comorphism, so it could be better moved e.g. to Common.Id.